### PR TITLE
feat: add more error metadata types

### PR DESCRIPTION
**Pull request by zhangyouxin**

### CODE REVIEW

File: src/components/ClaimSudt/index.tsx [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+import { LightGodwokenNotFoundError } from "../../light-godwoken/constants/error";
 
 export const ClaimSudt: React.FC = () => {
   const lightGodwoken = useLightGodwoken();
 
   const claimSudt = async () => {
     if (!lightGodwoken) {
-      notification.error({ message: "Please connect wallet first" });
-      throw new Error("LightGodwoken Not Found!");
+      const message = "Light Godwoken is not found";
```
Comment (sighWang) [inline src/components/ClaimSudt/index.tsx:10]: need to revert

File: src/light-godwoken/LightGodwokenV0.ts [hunk 2]
```diff
@@ -289,13 +304,13 @@ export default class DefaultLightGodwokenV0 extends DefaultLightGodwoken impleme
 
     debug("withdrawalRequestExtra:", withdrawalRequestExtra);
     // using RPC `submitWithdrawalRequest` to submit withdrawal request to godwoken
-    let txHash: unknown;
+    let txHash: any;
```
Comment (sighWang) [inline src/light-godwoken/LightGodwokenV0.ts:307]: avoid using any

File: src/light-godwoken/LightGodwokenV1.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { message } from "antd";
```
Comment (sighWang) [inline src/light-godwoken/LightGodwokenV1.ts:1]: removed

File: src/components/ClaimSudt/index.tsx (hunk 1 above)
Comment (zhangyouxin) [reply src/components/ClaimSudt/index.tsx:10]: Any details?

Comment (sighWang) [reply src/components/ClaimSudt/index.tsx:10]: we should let user know how to continue here,technical error like 'Light Godwoken is not found' is not a useful message for user

**Pull request merged**